src/spec.h

```diff
@@ -220,7 +220,8 @@ typedef uint16_t FieldSpecDedupeArray[SPEC_MAX_FIELDS];
   (Index_StoreFreqs | Index_StoreFieldFlags | Index_StoreTermOffsets | Index_StoreNumeric | \
    Index_WideSchema)
 
-#define INDEX_CURRENT_VERSION 25
+#define INDEX_CURRENT_VERSION 26
```

Comment:
why?

Reply:
Since we change the RDB serialization format.
We need to mark that with some version id. This is registered together with the `IndexSpec` type such that upon serialization, this is serialized as well and passed to our deserialization callback.